src/sentry/integrations/slack/metrics.py

```diff
@@ -0,0 +1,4 @@
+# metrics constants
+
+SLACK_ISSUE_ALERT_SUCCESS_DATADOG_METRIC = "sentry.integrations.slack.issue_alert.success"
+SLACK_ISSUE_ALERT_FAILURE_DATADOG_METRIC = "sentry.integrations.slack.issue_alert.failure"
```

Comment:
adding metrics, i thought to dump them here

Reply:
Love this pattern, thank you for adding and using constant!